example/deployment/AWS-EC2-S3/bash-scripts/query.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+export PALISADE_REST_CONFIG_PATH="/home/ec2-user/example/example-model/src/main/resources/configRest.json"
+
+if [[ $# -eq 1 ]]; then
+    java -cp /home/ec2-user/example/example-model/target/example-model-*-shaded.jar uk.gov.gchq.palisade.example.client.ExampleSimpleClient $1 s3a://palisade-ec2-demo/data "" | /home/ec2-user/example/deployment/bash-scripts/formatOutput.sh
+else
+    java -cp /home/ec2-user/example/example-model/target/example-model-*-shaded.jar uk.gov.gchq.palisade.example.client.ExampleSimpleClient $1 s3a://palisade-ec2-demo/data $2 | /home/ec2-user/example/deployment/bash-scripts/formatOutput.sh
+fi
+
+# Runs the demo example, start to finish, all users, all purposes
+# java -cp /home/ec2-user/example/example-model/target/example-model-*-shaded.jar uk.gov.gchq.palisade.example.runner.RestExample $1 | /home/ec2-user/example/deployment/bash-scripts/formatOutput.sh
```

Comment:
Are these comment lines here for editing purposes or are they dead code?

Reply:
they are there to show other ways that the query could be run

Reply:
OK that makes sense